ballerina/tests/pipe_test.bal

```diff
@@ -80,7 +80,7 @@ function testImmediateClose() returns error? {
     check pipe.produce("1", timeout = 5);
     check pipe.immediateClose();
     Error? actualValue = pipe.consume(5);
-    test:assertTrue(actualValue is ());
+    test:assertTrue(actualValue is Error);
```

Comment:
This is a breaking change right? Don't we have to update the major version?

Reply:
Here, it is just an updating a test case with a bug fix. Do we still need to update it to a major version?

Reply:
I think this should be considered as a breaking change since there could be user code which uses null check logic. At-least we should bump it to minor version which prevents ballerina automatically pulling patch version.